lib/id3c/cli/command/__init__.py

```diff
@@ -31,7 +32,7 @@
 LOG = logging.getLogger(__name__)
 
 CACHE_TTL = 60 * 60 * 24 * 365  # 1 year
-CACHE_SIZE = float("inf")       # Unlimited
+CACHE_SIZE = maxsize
```

Comment:
It's good to fix this type mismatch now that mypy detects and complains about it.

The rationale for using `Inf` in the first place was that cachetools only does comparisons like `<` that are supported between `int` and `float`, and also that [cachetools itself (still!) uses `Inf`](https://github.com/tkem/cachetools/blob/f9704abaaba7cd4ce807aed3d968fb391107d8c6/cachetools/func.py#L41) to get an unbounded cache. :-)